src/main/java/io/ebeaninternal/dbmigration/DefaultDbMigration.java

```diff
@@ -222,6 +224,15 @@ public void addDatabasePlatform(DatabasePlatform databasePlatform, String prefix
   @Override
   public String generateMigration() throws IOException {
 
+
+    // configure the platform
```

Comment:
Yeah ok ... anything DDL / migration related really need to be in PlatformConfig to support this